tests/e2e_http.py

```diff
@@ -26,6 +26,9 @@ def run(args):
             hosts, args.build_dir, args.debug_nodes, args.perf_nodes, pdb=args.pdb
         ) as network:
             primary, _ = network.start_and_join(args)
+
+            time.sleep(3)
```

Comment:
Why?

Reply:
It was already there for the http-specific code (https://github.com/microsoft/CCF/blob/master/tests/infra/ccf.py#L516). I just moved it out of `ccf.py`.

Reply:
As discussed, it seems that it is necessary as the HTTP client does not have a retry logic yet. Will be fixed in a later PR.